bambi/priors.py

```diff
@@ -248,9 +248,7 @@ def _get_slope_stats(self, exog, predictor, sigma_corr, full_mod=None, points=4)
             null = [
                 GLM(
                     endog=self.model.y.data, exog=exog, family=self.model.family.smfamily()
-                ).fit_constrained(
-                    str(exog.columns[i]) + "=" + str(val), start_params=None
-                )
+                ).fit_constrained(str(exog.columns[i]) + "=" + str(val), start_params=None)
```

Comment:
`start_params` is None by default we can omit it.

Reply:
You're right! :man-facepalming: